clients/java/src/main/java/io/zeebe/client/impl/oauth/OAuthCredentialsProvider.java

```diff
@@ -72,9 +71,12 @@ public void applyCredentials(final Metadata headers) throws IOException {
       loadCredentials();
     }
 
-    headers.put(
-        HEADER_AUTH_KEY,
-        String.format("%s %s", credentials.getTokenType(), credentials.getAccessToken()));
+    String type = credentials.getTokenType();
+    if (!type.isEmpty()) {
+      type = Character.toUpperCase(type.charAt(0)) + type.substring(1);
```

Comment:
If `type == 'c'`, I think this will throw an out of bounds exception when doing `type.substring(1)`?

Reply:
No, only if the index (1) was larger than the length (which we check in the line before)

Reply:
:+1: